sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -1403,9 +1403,10 @@ func (host *nodeLanguageHost) RunPlugin(
 		return err
 	}
 
-	nodeargs = append(nodeargs, req.Info.EntryPoint)
```

Comment:
I don't think EntryPoint is ever anything but "." today, but if we add "main" support to PulumiPlugin.json we'd probably need to use it in here somewhere.